com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1402,29 +1402,30 @@ public Void call() throws Exception {
      *  Options from the `game.project` file that may affect build outputs.
      */
     private static class GameProjectBuildOption {
-        public String inputOption, outputOption, propertyCategory, propertyKey, appManifestSymbol;
+        public String inputOption, outputOption, propertyCategory, propertyKey;
+        public List<String> appManifestSymbols;
         /**
          * @param inputOption        Option that may be used with Bob.
          * @param outputOption       How the option will be saved in project options using project.setOption() for future use.
          * @param propertyCategory   Category in the `game.project` file.
          * @param propertyKey        Key in the `game.project` file.
-         * @param appManifestSymbol  A symbol from appManifest that makes this option true.
+         * @param appManifestSymbols Symbols from appManifest that makes this option true.
          */
-        public GameProjectBuildOption(String inputOption, String outputOption, String propertyCategory, String propertyKey, String appManifestSymbol) {
+        public GameProjectBuildOption(String inputOption, String outputOption, String propertyCategory, String propertyKey, List<String> appManifestSymbols) {
             this.inputOption = inputOption;
             this.outputOption = outputOption;
             this.propertyCategory = propertyCategory;
             this.propertyKey = propertyKey;
-            this.appManifestSymbol = appManifestSymbol;
+            this.appManifestSymbols = appManifestSymbols;
         }
     }
 
     public void configurePreBuildProjectOptions() throws IOException, CompileExceptionError {
         List<GameProjectBuildOption> options = new ArrayList<>();
-        options.add(new GameProjectBuildOption("debug-output-spirv", "output-spirv", "shader","output_spirv","GraphicsAdapterVulkan"));
-        options.add(new GameProjectBuildOption("debug-output-hlsl", "output-hlsl", "shader","output_hlsl","GraphicsAdapterDX12"));
-        options.add(new GameProjectBuildOption("debug-output-wgsl", "output-wgsl", "shader","output_wgsl","GraphicsAdapterWebGPU"));
-        options.add(new GameProjectBuildOption("debug-output-glsl", "output-glsl", "shader","output_glsl","GraphicsAdapterOpenGL"));
+        options.add(new GameProjectBuildOption("debug-output-spirv", "output-spirv", "shader","output_spirv",List.of("GraphicsAdapterVulkan")));
+        options.add(new GameProjectBuildOption("debug-output-hlsl", "output-hlsl", "shader","output_hlsl",List.of("GraphicsAdapterDX12")));
+        options.add(new GameProjectBuildOption("debug-output-wgsl", "output-wgsl", "shader","output_wgsl",List.of("GraphicsAdapterWebGPU")));
+        options.add(new GameProjectBuildOption("debug-output-glsl", "output-glsl", "shader","output_glsl",List.of("GraphicsAdapterOpenGL", "GraphicsAdapterOpenGLES")));
```

Comment:
Change from String to List of string because glsl is needed by OpenGL and OpeGLES.

Reply:
it makes sense